### PR TITLE
Updated roadmap for Q1 2021

### PR DESCRIPTION
**Pull request by christophcrichter**

### CODE REVIEW

File: docs/roadmap.md [hunk 1]
```diff
@@ -2,28 +2,25 @@
...
+The Blue/Green deployment strategy is a popular way of performing zero downtime deployments while retaining guarantees about the integrity of the whole environment. It works by deploying to a secondary (Green) environment and once this environment has been validated, cut the traffic from the current (Blue) environment to the new Green environment. The process is repeated, this time from Green to Blue at the next deployment. Compared with the zero-downtime “rolling update” strategy that Humanitec defaults to, a Blue/Green deployment strategy provides more control to operations and development teams deploying applications.
 
-## Q1 2021 and beyond
+## Q2 2021 and beyond
 
 The following features are in our backlog and planned for implementation from Q4 2020 onwards. They are subject to change based on additional feedback we will gather until then.
```
Comment (chrishumanitec) [inline docs/roadmap.md:23]: Canary deployments is missing

File: docs/roadmap.md [hunk 2]
```diff
@@ -2,28 +2,25 @@
 
 This document provides a high-level view on our product roadmap for this and upcoming quarters. It also lists roadmap items of past quarters.
 
-## Q4 2020
+## Q1 2021
 
-### Volumes
```
Comment (chrishumanitec) [inline docs/roadmap.md:7]: Why are volumes being removed?

File: docs/roadmap.md [hunk 3]
```diff
@@ -44,6 +41,20 @@ Humanitec is sitting on a large amount of data and so has the potential to provi
 
 ## Past Quarters
 
+### Q4 2020
+
+#### Volumes
```
Comment (chrishumanitec) [inline docs/roadmap.md:46]: Should be in Q1

**Review by chrishumanitec — CHANGES_REQUESTED** [review]
Volumes is missing from  Q1
Canary deployments is missing from future items.

File: docs/roadmap.md (hunk 1 above)
Comment (christophcrichter) [reply docs/roadmap.md:23]: Will add it.

File: docs/roadmap.md (hunk 3 above)
Comment (christophcrichter) [reply docs/roadmap.md:46]: Will push them up again

File: docs/roadmap.md (hunk 2 above)
Comment (christophcrichter) [reply docs/roadmap.md:7]: Moved back again.

**Review by chrishumanitec — APPROVED** [review]
LGTM

**Pull request merged**